### PR TITLE
adds terrorist room to maint + replaces mini syringegun with functionally the same mk22

### PR DESCRIPTION
**Pull request by ghost**
http://vignette4.wikia.nocookie.net/metalgear/images/1/1e/Mk22_1-300x170.png/revision/latest?cb=20120424225942

looks cooler, also memes added to maint 
http://puu.sh/mZiaz/9386609bf4.png
http://puu.sh/mZiLu/07cc83e07d.png

### CODE REVIEW

**Comment by Spacedong** [comment]
wait whats going on here is it an actual dart gun or does it just have the foam pistol sprite

**Comment by ghost** [comment]
:+1:

**Comment by ghost** [comment]
that picture of the map shows the terrorist room.
the other one is the gun resprited/name/desc

**Comment by Kokojos** [comment]
No. A functionnal gun each time each round in the same position? Absolutely not. Once it's known everybody will rush to it 24h/24.

Even if it's a syringe tranquiliser.

**Comment by Spacedong** [comment]
@kokojos its a foam dart gun, he just renamed the dart pistol traitor item to something more tacticool, he didnt actually place it on the map

**Comment by ghost** [comment]
sry koko but PLSSS look at the entire thingie!!!!
i just reskinned the traitor item,then added a completely seperate terrorist room which introduces a smoke grenade and a balaclava.

**Comment by Kokojos** [comment]
Welp. Not obvious on first sight, but my bad.

**Comment by Nokington** [comment]
yes

**Comment by ghost** [comment]
and if a nigga get an attitude

**Comment by danhibiki** [comment]
Yes

**Comment by nalarmg** [comment]
Conflict.

File: code/modules/projectiles/guns/syringe_gun.dm [hunk 1]
```diff
@@ -80,12 +80,12 @@
 	max_syringes = 6
 
 /obj/item/weapon/gun/syringe/syndicate
-	name = "dart pistol"
-	desc = "A small spring-loaded sidearm that functions identically to a syringe gun."
-	icon_state = "syringe_pistol"
+	name = "Mk22"
+	desc = "A small spring-loaded 9mm tranquilizer sidearm."
```
Comment (Carbonhell) [inline code/modules/projectiles/guns/syringe_gun.dm:84]: I don't get it,shouldn't the desc be the same of the one inside uplink_item, plus what the fuck it's not spring loaded,it's a mk22

**Comment by TDcoolguy300** [comment]
Appears that the DMI is having a merge conflict, not sure.

**Comment by Nebyab** [comment]
sure, fix it and its good

**Comment by nalarmg** [comment]
Conflicted for over a week, no visible effort or interest in fixing it. Closing. Feel free to reopen/submit a new PR if there's further interest in this.

**Comment by ghost** [comment]
ya i kinda had no internet so ill fix this real guick

**Pull request closed without merging**